docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -41,10 +41,7 @@ The IBM FHIR Server implements the HL7 FHIR HTTP API and supports the full set o
 This FHIR server is intended to be a common component for providing FHIR capabilities within health services and solutions.
 
 ## 1.1 Recent updates
-View information about recent changes that were made to this document. For more information about changes that were made to the FHIR server codebase, see the corresponding release notes from the GitHub Releases tab.
-
-### Release 4.0
-* Initial release of the IBM FHIR Server for HL7 FHIR R4
+View information about recent changes that were made to this document. For more information about changes that were made to the FHIR server codebase, see the corresponding release notes from the [GitHub Releases tab](https://github.com/IBM/FHIR/releases).
```

Comment:
I've been wondering if we should just remove this section...we havn't been good about updating it when we update the docs.

Reply:
Do you want me to remove the Recent Updates section?

Reply:
I'm fine with either:
A. remove it and be done; or
B. keep it, try to update it for 4.1 and 4.2, and then try keeping it up-to-date going forward

Reply:
We selected (A)